plugin/pkg/plugins.go

```diff
@@ -96,7 +97,6 @@ func AllPluginNames() []string {
 		PluginNameOpenIDConnectPreset,               // OpenIDConnectPreset
 		PluginNameClusterOpenIDConnectPreset,        // ClusterOpenIDConnectPreset
 		PluginNameCustomVerbAuthorizer,              // CustomVerbAuthorizer
-		PluginNameShootVPAEnabledByDefault,          // ShootVPAEnabledByDefault
```

Comment:
The plugins returned by the `AllPluginNames` func are ordered. See https://github.com/gardener/gardener/blob/485c25124ea536e4610c627109017dd34d434921/plugin/pkg/plugins.go#L114-L116 and https://github.com/gardener/gardener/blob/42f1081c4f9ac7b42c9c935920d8a937acb0cc93/cmd/gardener-apiserver/app/gardener_apiserver.go#L141.

Hence, I would suggest to preserve the order of the admission plugin in this list to do not cause any side effect. I don't have a concrete case in mind but this change looks unrelated to what we want to achieve with this PR.

For the `DefaultOnPlugins` func, we have a set and we can follow alphabetic order there.